sdk/include/reactos/undocuser.h

```diff
@@ -204,6 +204,20 @@ BOOL WINAPI PaintMenuBar(HWND hWnd, HDC hDC, ULONG left, ULONG right, ULONG top,
 #define DrawCaptionTemp DrawCaptionTempA
 #endif
 
+//
+// Hard error balloon package
+//
+typedef struct _BALLOON_HARD_ERROR_DATA
+{
+    DWORD cbHeaderSize;
```

Comment:
why "header"? Just "cbSize" should be enough :P

Reply:
because it doesn't include the size of the strings